lib/tasks/stop_data_convert_and_import.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'csv'
+namespace :bus_stops do
+  task :import_data, [:csv_file] => :environment do |_, args|
+    csv = CSV.parse(File.read(args[:csv_file]), headers: true)
+    csv.each do |row|
+      hash = row.to_hash
```

Comment:
I think it's a good idea to separate the data we're reading from the data we're writing - that is, to use two separate data structures. One that we don't touch, which is our input data, and one which starts out empty and which we do modify, which becomes the attributes.

Reply:
I understand what you're saying but could you please explain why this is necessary? If we're not saving over the CSV why does it matter how we modify it in the task?

Reply:
Clarified in person that it's more about the code readability than its function necessarily.